benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_subdirectory(jsonschema)
+
+add_custom_target(run_benchmark
+    COMMAND sourcemeta_jsontoolkit_benchmark_jsonschema
+    DEPENDS sourcemeta_jsontoolkit_benchmark_jsonschema
+    WORKING_DIRECTORY ${CMAKE_PROJECT_DIR}
```

Comment:
I don't think you need to specify a working directory here, as all the benchmarks we will use are working directory agnostic? We can let CMake use the current binary directory as a default